deployment/ccip/changeset/test_assertions.go

```diff
@@ -259,6 +259,34 @@ func (c *commitReportTracker) allCommited(sourceChainSelector uint64) bool {
 	return true
 }
 
+func ConfirmMultipleCommits(
+	t *testing.T,
+	chains map[uint64]deployment.Chain,
+	state map[uint64]CCIPChainState,
+	startBlocks map[uint64]*uint64,
+	enforceSingleCommit bool,
+	expectedSeqNums map[SourceDestPair]ccipocr3.SeqNumRange,
+) error {
+	errGrp := &errgroup.Group{}
+
+	for sourceDest, seqRange := range expectedSeqNums {
+		errGrp.Go(func() error {
```

Comment:
might run into overrides, better to create local scope for `sourceDest` and `seqRange`